src/styles/_catalog.scss

```diff
@@ -1,22 +1,37 @@
+$cardWidth: 200px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(4, $cardWidth);
+  gap: 48px;
+
+  justify-content: center;
+
+  @media (max-width: 1023.5px) {
+    & {
+      grid-template-columns: repeat(3, $cardWidth);
+    }
+  }
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (max-width: 767.5px) {
+    & {
+      grid-template-columns: repeat(2, $cardWidth);
+    }
```

Comment:
It's extra
```suggestion
      grid-template-columns: repeat(2, $cardWidth);
```

Reply:
Thanks for review, corrected everything)